core/dbt/events/functions.py

```diff
@@ -43,12 +42,18 @@ def setup_event_logger(log_path: str, log_format: str, use_colors: bool, debug:
 
 
 def _get_stdout_config(log_format: str, debug: bool, use_colors: bool) -> LoggerConfig:
+    flags = get_flags()
     fmt = LineFormat.PlainText
     if log_format == "json":
         fmt = LineFormat.Json
     elif debug:
         fmt = LineFormat.DebugText
     level = EventLevel.DEBUG if debug else EventLevel.INFO
+    # We don't have access to these values when we need to setup the default stdout logger!
```

Comment:
@peterallenwebb I don't think we'll be able to solve https://github.com/dbt-labs/dbt-core/issues/6847 in the same way you did https://github.com/dbt-labs/dbt-core/pull/6886, because we now need a click Context to access `flags.QUIET` (even if it's passed in on the CLI), and we don't have access to that as early as we used to.

That approach was always a bit opportunistic, so I don't feel like this is a huge deal. Open to ideas for sure.